src/features/dwn/dwn-service.ts

```diff
@@ -1,43 +1,42 @@
 import {
   Dwn,
-  EventLogLevel,
-  MessageStoreLevel,
-  DataStoreLevel,
   RecordsWrite,
   DataStream,
   DidKeyResolver,
   RecordsQuery,
 } from "@tbd54566975/dwn-sdk-js";
 import { MemoryLevel } from "memory-level";
-import { RNLevel } from "./rn-level";
 import {
   DataStoreSql,
   EventLogSql,
   MessageStoreSql,
   SqliteDialect,
 } from "@tbd54566975/dwn-sql-store";
 import { SqliteDatabase } from "./sqlite-config";
+import {
+  MessageStoreLevel,
+  DataStoreLevel,
+  EventLogLevel,
+} from "@tbd54566975/dwn-sdk-js/dist/esm/src/index-stores";
```

Comment:
`MessageStoreLevel`, `DataStoreLevel`, and `EventLogLevel` are no longer exported by default in the `dwn-sdk-js` package. Instead, they are exposed via the `stores` export [here](https://github.com/TBD54566975/dwn-sdk-js/blob/main/package.json#L57). 

React Native currently doesn't have named export support (but it is coming soon), so I'm just manually referencing that file here. Open to other options if there's a better way to do this for the interim!

Reply:
We gotta get these exports added back. There was no great reason to remove them, as we needed all of our polyfills and bundler configs no matter what.